code/datums/id_trim/jobs.dm

```diff
@@ -824,6 +855,8 @@
 		ACCESS_CMO,
 		)
 	job = /datum/job/paramedic
+	honorifics = list("EMT", "Doctor", "Dr.")
```

Comment:
paramedics aren't doctors

Reply:
Gonna sidestep a bunch of RPR9 discourse and go with "google says paramedics aren't doctors" instead